wolfcrypt/test/test.c

```diff
@@ -25283,10 +25283,85 @@ static int crypto_ecc_sign(const uint8_t *key, uint32_t keySz,
 
     return ret;
 }
+#endif /* HAVE_ECC_SIGN && HAVE_ECC_VERIFY */
 
-static int ecc_test_nonblock(WC_RNG* rng)
+#ifdef HAVE_ECC_DHE
+/* 
+ * Currently we only have support for P-256/384 for non-blocking key gen and
+ * shared secret gen.
+ */
+static int ecc_test_nonblock_dhe(WC_RNG* rng)
```

Comment:
This only runs for the largest enabled curve. Can we run the SP non-blocking test for 256, 384 and 521?